content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Why is Redfish® different from other REST APIs - Part 1
+date: 2024-02-19T16:20:51.657Z
+author: François Donzé
+authorimage: /img/fdz-photoprofile.png
+thumbnailimage: ""
+disable: false
+tags:
+  - Redfish
+  - ilo-restful-api
+---
+## Redfish® brief recap
+
+Redfish® is a low level management RESTful API standardized by the Distributed Management Task Force (<a href="https://dmtf.org" target="_blank">DMTF</a>) consortium. DMTF publishes several standards related to server management, like the Common Information Model (CIM) implemented as OpenPegasus on Linux and Windows Management Instrumentation (WMI) on Microsoft Windows.
+
+Redfish® is born in August 2015. As of today, more than <a href="https://dmtf.org/about/list" target="_blank">sixty members</a> contribute to its development: computer makers, storage and network device manufacturers and software developers. A funny story says that "Redfish®" comes from the name of a restaurant near the Compaq drive in Houston, Texas, where initiators of the project were meeting before DMTF took over the entire project.
+
+The main goal of Redfish® is to replace the Intelligent Platform Management Interface (<a href="https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-home.html" target="_blank">IMPI</a>) originally created by Intel®. In a nutshell, Redfish® is able to monitor, configure and perform actions (i.e. power-on/off) on remote (out-of-band) or local (in-band) servers. Also, it provides an event subscription mechanism that can replace the Simple Network Management Protocol (SNMP).
+
+The adoption of this standard across the industry has been very quick and this article is an attempt to explain some of the specificities that have contributed to this success.
+
+## Redfish® specificities
+
+Redfish® is classified as a RESTful API as all the fundamentals are present, like a service/client model, HTTPS JSON formatted requests transferring representational states from and to identified end points. However, several specificities are unique, like the separation of the protocol and the data model and the possibility to enhance the data model with Original Equipment Manufacturer (OEM) extensions.
```

Comment:
Shouldn't it be a client/server model?

Where you have the second comma in the first sentence, replace it with the word "and" so it reads:
"...all the fundamentals are present, like a client/server model and HTTPS JSON Formatted requests that transfer representational states from and to identified end points."